src/Facil.Generator/Render.fs

```diff
@@ -301,14 +301,19 @@ let private renderProcOrScript (cfg: RuleSet) (tableDtos: TableDto list) (execut
             ""
 
             $"let getItem (reader: SqlDataReader){getItemReturnTypeExpr} ="
+            yield! indent [
+              for c in cols do
+                if c.IsNullable then
+                  $"let ``value_{c.Name.Value}`` = if reader.IsDBNull ``ordinal_{c.Name.Value}`` then {outOptionNone} else reader.{c.TypeInfo.SqlDataReaderGetMethodName} ``ordinal_{c.Name.Value}`` |> {outOptionSome}"
```

Comment:
Is there any point to prefixing with `value_`? If not, I suggest removing the prefix.

Reply:
`value_` has been removed